splunk-forwarder/main.tf

```diff
@@ -0,0 +1,111 @@
+resource "random_string" "unique_identfier" {
+  length  = 8
+  special = false
+}
+
+resource "aws_iam_role" "lambda_role" {
+  name = "lambda_role_${random_string.unique_identfier.result}"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": "sdsd"
```

Comment:
Looks like a placeholder :)

Reply:
Forgot, in a brief moment, that the repo was open source :-)